.github/workflows/release.yml

```diff
@@ -5,7 +5,10 @@ on:
   push:
     # Sequence of patterns matched against refs/tags
     tags:
-      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "v[0-9]+\\.[0-9]+\\.[0-9]+" # Official release version tags e.g. v2.0.5
+      - "v[0-9]+\\.[0-9]+\\.[0-9]+-rc[0-9]+" # Release candidate tags e.g. v1.0.3-rc4
```

Comment:
```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+" # Official release version tags e.g. v2.0.5
      - "v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+" # Release candidate tags e.g. v1.0.3-rc4
```
Slashes needed?

Reply:
by putting the slashes in the regexp is more exact. `\\.` means actualy period. In regexp `.` means 'any character'.

Reply:
_claps in semver_